packages/babel-traverse/README.md

```diff
@@ -1 +1,30 @@
 # babel-traverse
+The Babel Traverse module maintains the overall tree state, and is responsible for replacing, removing, and adding nodes.
```

Comment:
Just for consistency and website parsing, let's add a newline and prefix with ">"